mtp_api/apps/performance/management/commands/update_performance_platform.py

```diff
@@ -1,3 +1,7 @@
+"""
+NB: GOV.UK have retired the performance platform
```

Comment:
worth deleting this whole file? (and `mtp_api/apps/performance/updaters.py`) or are they still used somehow?

Reply:
as i say, i'm only suggesting removing the scheduling of these updates (which raise errors) rather than deleting the updaters altogether at this point.